src/components/input-dropdown/index.jsx

```diff
@@ -44,29 +57,44 @@ class InputDropdown extends React.PureComponent {
         } = this.state
 
         const classes = classNames('td-input-dropdown', className)
+        const dropDownClasses = 'td-input-dropdown__dropdown'
+        const listClasses = 'td-input-dropdown__list'
 
         return (
             <div className={classes}>
-                <Input 
-                    className="td-input-dropdown__selection"
+                <Input className="td-input-dropdown__selection"
```

Comment:
When the markup gets complex, if we have all our classes at the top -- it becomes easier to find/modify them.

Helps with developer experience in my opinion